app/controllers/forms_controller.rb

```diff
@@ -78,7 +78,7 @@ def show
 
         # xml style defaults to odk but can be specified via query string
         @style = params[:style] || 'odk'
-
+        setup_condition_computer
```

Comment:
@meltheadorable I made the condition computer available here in the controller to be used in the odk views, in case it's relevant to your web view work.